backend/usecase/signin.go

```
@@ -0,0 +1,61 @@
+	return &SignIn{db, tokenGenerator}
+}
+
+type SignInInput struct {
+	Email    string `json:"email" binding:"required"`
+	Password string `json:"password" binding:"required"`
+}
+
+func (s *SignIn) SignIn(c *gin.Context) {
+	var input SignInInput
+	if err := c.ShouldBindJSON(&input); err != nil {
+		c.Error(api_error.NewBadRequestError(err.Error(), err))
```

golangci-lint

Error return value of `c.Error` is not checked

(errcheck)